src/setup_node_env/harden/console.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// From https://www.ascii-code.com/characters/control-characters,
+// but explicitly allowing the range \u0008-\u000F (line breaks, tabs, etc.)
+var CONTROL_CHAR_REGEXP = new RegExp('[\\u0000-\\u0007\\u0010-\\u001F]', 'g');
+
+var unsafeConsole = {
+  // debug: console.debug.bind(console),
+  // error: console.error.bind(console),
+  // info: console.info.bind(console),
+  log: console.log.bind(console),
+  // trace: console.trace.bind(console),
+  // warn: console.warn.bind(console),
+};
+
+module.exports = { unsafeConsole };
+
+console.log = function () {
+  var args = Array.from(arguments);
+  var cleanedArgs = args.map(function (element) {
```

Comment:
I wasn't really sure if this was the right way to handle multiple arguments. One function prototype of `console.log` is (message?: any, ...optionalParams: any[]). I assumed we'd want to attempt to clean the optional params.

Reply:
So this may be a bit tricky. Technically, `console.log` calls `util.inspect` under the hood for parameters that aren't plain strings. So if we want to preserve this behavior, we may need to do some filtering and not blindly call `escapeControlChars` for all args (given this function coerce everything to string)